scripts/install.sh

```diff
@@ -0,0 +1,174 @@
+    echo "[ERROR] Please set environment variable INVENIO_USER_EMAIL before runnning this script."
+    echo "[ERROR] Example: export INVENIO_USER_EMAIL=info@inveniosoftware.org"
+    exit 1
+fi
+if [ -v ${INVENIO_USER_PASS} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_USER_PASS before runnning this script."
+    echo "[ERROR] Example: export INVENIO_USER_PASS=uspass123"
+    exit 1
+fi
+if [ -v ${INVENIO_POSTGRESQL_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_POSTGRESQL_HOST=192.168.50.11."
+    exit 1
+fi
+if [ -v ${INVENIO_POSTGRESQL_DBNAME} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_DBNAME before runnning this script."
+    echo "[ERROR] Example: INVENIO_POSTGRESQL_DBNAME=invenio3"
+    exit 1
+fi
+if [ -v ${INVENIO_POSTGRESQL_DBUSER} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_DBUSER before runnning this script."
+    echo "[ERROR] Example: INVENIO_POSTGRESQL_DBUSER=invenio3"
+    exit 1
+fi
+if [ -v ${INVENIO_POSTGRESQL_DBPASS} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_POSTGRESQL_DBPASS before runnning this script."
+    echo "[ERROR] Example: INVENIO_POSTGRESQL_DBPASS=dbpass123"
+    exit 1
+fi
+if [ -v ${INVENIO_REDIS_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_REDIS_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_REDIS_HOST=192.168.50.12."
+    exit 1
+fi
+if [ -v ${INVENIO_ELASTICSEARCH_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_ELASTICSEARCH_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_ELASTICSEARCH_HOST=192.168.50.13."
+    exit 1
+fi
+if [ -v ${INVENIO_RABBITMQ_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_RABBITMQ_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_RABBITMQ_HOST=192.168.50.14."
+    exit 1
+fi
+if [ -v ${INVENIO_WORKER_HOST} ]; then
+    echo "[ERROR] Please set environment variable INVENIO_WORKER_HOST before runnning this script."
+    echo "[ERROR] Example: export INVENIO_WORKER_HOST=192.168.50.15."
+    exit 1
+fi
+
+# load virtualenvrapper:
+source $(which virtualenvwrapper.sh)
+
+# sphinxdoc-create-virtual-environment-begin
+mkvirtualenv ${INVENIO_WEB_VENV}
+cdvirtualenv
+mkdir -p src
+cd src
+# sphinxdoc-create-virtual-environment-end
+
+# quit on errors and unbound symbols:
+set -o errexit
+set -o nounset
+
+# sphinxdoc-install-invenio-full-begin
+# FIXME we pin older version that works:
+pip install invenio-records==v1.0.0a3
+# FIXME we need to install PostgreSQL DB connect option explicitly:
+pip install invenio-db[postgresql]
+# now we can install full Invenio:
+pip install invenio[full] --pre
+# sphinxdoc-install-invenio-full-end
+
+# sphinxdoc-create-instance-begin
+inveniomanage instance create mysite
```

Comment:
Perhaps we can reuse the virtual environment name here, so that people can type blindly `workon zenodo && cdvirtualenv src/zenodo` without too much thinking, for simplicity's sake.